cypress/integration/hosts/host_filtering.ts

```diff
@@ -104,170 +81,57 @@ const filterTests = [
-      cy.dataCy(distroFilterIconDataCy)
-        .should("be.visible")
-        .should("not.be.disabled");
-    });
-
-    it("Filters hosts with input value when Enter key is pressed", () => {
-      cy.dataCy(distroFilterIconDataCy).scrollIntoView().click();
-      cy.dataCy(`${distroFilterIconDataCy}-wrapper`).within(() => {
-        cy.dataCy("input-filter").type("centos6-perf").type("{enter}");
-      });
-
-      cy.get(tableRow).each(($el, index) =>
-        cy
-          .wrap($el)
-          .contains(
-            [
-              "16326bd716fd4ad5845710c479c79e86c66b61bcef8ebbe7fc38dfc36fab512e",
-              "1694cfe1eac28b3316339f6276021afcb2a07bcd21a266405835fd039557ea2d",
-              "4b332e12790a585a0c7cbaf1650674f408117cf6134679c9e5f2e96cadd07923",
-              "6e331e02aaaebba422d1f1d2dbd3e64f01776b84c68c672ea680e4b81b0719bb",
-              "7f909d47566126bd39a05c1a5bd5d111c2e68de3830a8be414c18c231a47f4fc",
-              "a99b50cd37b012c53db7207e4ba8b52989aefab551176c07962cea979abcc479",
-              "b700d10f21a5386c827251a029dd931b5ea910377e0bb93f3393b17fb9bdbd08",
-              "build10.ny.cbi.10gen",
-              "build10.ny.cbi.10gen.c",
-              "build10.ny.cbi.10gen.cc",
-              "c04d193c4de174376167746bc268426a4085bffb364c4740e0564ca3eeee6875",
-              "i-0a5b8aa85c469e35d",
-            ][index]
-          )
-      );
-    });
-
-    it("Trims the whitespace from filter input values", () => {
-      cy.dataCy(distroFilterIconDataCy).scrollIntoView().click();
-
-      cy.dataCy(`${distroFilterIconDataCy}-wrapper`).within(() => {
-        cy.dataCy("input-filter")
-          .type("      centos6-perf     ")
-          .type("{enter}");
-      });
-
-      cy.get(tableRow).each(($el, index) =>
-        cy
-          .wrap($el)
-          .contains(
-            [
-              "16326bd716fd4ad5845710c479c79e86c66b61bcef8ebbe7fc38dfc36fab512e",
-              "1694cfe1eac28b3316339f6276021afcb2a07bcd21a266405835fd039557ea2d",
-              "4b332e12790a585a0c7cbaf1650674f408117cf6134679c9e5f2e96cadd07923",
-              "6e331e02aaaebba422d1f1d2dbd3e64f01776b84c68c672ea680e4b81b0719bb",
-              "7f909d47566126bd39a05c1a5bd5d111c2e68de3830a8be414c18c231a47f4fc",
-              "a99b50cd37b012c53db7207e4ba8b52989aefab551176c07962cea979abcc479",
-              "b700d10f21a5386c827251a029dd931b5ea910377e0bb93f3393b17fb9bdbd08",
-              "build10.ny.cbi.10gen",
-              "build10.ny.cbi.10gen.c",
-              "build10.ny.cbi.10gen.cc",
-              "c04d193c4de174376167746bc268426a4085bffb364c4740e0564ca3eeee6875",
-              "i-0a5b8aa85c469e35d",
-            ][index]
-          )
-      );
-    });
-
-    filterTests.forEach(({ param, filterIconDataCy, filterValue }) => {
+  filterTests.forEach(
+    ({ param, filterIconDataCy, filterValue, filterUrlParam, expectedIds }) => {
       it(`Filters hosts using table filter dropdowns for ${param}`, () => {
```

Comment:
Seems like the other tests in this file disappeared; did you intend to delete them all except for this one?

Reply:
yes i ended up incorporating them into the test. :) Theres an assertion now to check if the url matches what we expect